client/account-recovery/lost-password/index.jsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { findDOMNode } from 'react-dom';
+import { localize } from 'i18n-calypso';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import support from 'lib/url/support';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormLabel from 'components/forms/form-label';
+import FormInput from 'components/forms/form-text-input';
+
+export class LostPassword extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			isSubmitting: false,
+		};
+	}
+
+	onSubmit = () => {
+		const userLogin = findDOMNode( this.refs.user_login ).value;
```

Comment:
Done in https://github.com/Automattic/wp-calypso/pull/8955/commits/a712370dabb06914d74da9e7f2bd317b10b6c3db